plugins/versionpress/tests/Workflow/CloneMergeTest.php

```diff
@@ -143,8 +143,11 @@ public function dateModifiedMergesAutomatically()
             ['field' => 'post_modified_gmt']
         );
 
-        $this->assertEquals($clonedModifiedDate, $modifiedDate);
-        $this->assertEquals($clonedModifiedDateGmt, $modifiedDateGmt);
+        // post_modified and post_modified_gmt are explicitly set to "now"
+        // see commit ff504f77
+        // therefore, the dates won't be exactly the same due to processing time and the sleep(1) above
+        $this->assertEquals(new DateTime($clonedModifiedDate), new DateTime($modifiedDate), '', 5);
+        $this->assertEquals(new DateTime($clonedModifiedDateGmt), new DateTime($modifiedDateGmt), '', 5);
```

Comment:
I'm not sure this is correct. The point of the merge driver is to make both dates the same after merge, and the merge happens above on [line 130](https://github.com/versionpress/versionpress/blob/master/plugins/versionpress/tests/Workflow/CloneMergeTest.php#L130). Here's a snippet from the merge driver:

https://github.com/versionpress/versionpress/blob/f5c6c2e624be6e49ac7afc052c219974fe5e4686/plugins/versionpress/src/Git/merge-drivers/ini-merge.php#L36-L41

So at this point, `$clonedModifiedDate` should be equal to `$modifiedDate`. Is that correct, @JanVoracek?

Reply:
It looks like that behavior changed in ff504f77 so the date should be "now" and not whatever it was before

Reply:
@borekb: Yes, it is correct, I guess. And notice that test works for me when PHP implementation of merge driver is forced. But test is using [Bash implementation](https://github.com/versionpress/versionpress/blob/f5c6c2e624be6e49ac7afc052c219974fe5e4686/plugins/versionpress/src/Git/merge-drivers/ini-merge.sh) by default and there is probably some hidden problem.